appendix.ru.md

```diff
@@ -6,26 +6,26 @@ lang: ru
 
 [0]: https://selamjie.medium.com/remove-richard-stallman-appendix-a-a7e41e784f88
 
-Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написал Селам Ж. [1] Особенно пугает то, как Столлман относится к обвинениям в том, что Марвин Мински изнасиловал одну из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джуффре), сказав: «но наиболее вероятный сценарий событий состоит в том, что она представила себя полностью готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «нападение предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.». [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Мински, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
+Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написала Селам Ж. [1] Особенно пугает то, как Столлман относится к обвинениям в том, что Марвин Минский изнасиловал одну из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джюффре), сказав: «но наиболее вероятный сценарий событий состоит в том, что она представила себя полностью готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «нападение предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.». [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Марвина, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
```

Comment:
```suggestion
Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написала Селам Ж. [1] Особенно пугает то, как Столлман относится к обвинениям в том, что Марвин Минский изнасиловал одну из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джюффре), сказав: «но наиболее вероятный сценарий событий состоит в том, что она представила себя полностью готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «нападение предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.» [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Марвина, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
```

Reply:
Already fixed

Reply:
No, it isn’t:
![Screenshot from 2021-03-31 19-21-08](https://user-images.githubusercontent.com/309253/113185069-4789fb80-9256-11eb-9608-c2e64d19ffcb.png)

(The change is about removing the second full stop.)

Reply:
It is now.

Reply:
Sorry, i'll remove this dot.

Reply:
Fixed.